omaha/base/constants.h

```diff
@@ -153,13 +153,13 @@ const TCHAR* const kTemporaryFilenamePrefix = _T("gup");
 // Omaha's app ID
 //
 // TODO(omaha): Rename all of these "Omaha".
-#define GOOPDATE_APP_ID _T("{430FD4D0-B729-4F61-AA34-91526481799D}")
+#define GOOPDATE_APP_ID _T("{1E972C4E-6353-4467-BF19-94DA1127DEAC}")
 const TCHAR* const kGoogleUpdateAppId = GOOPDATE_APP_ID;
-const GUID kGoopdateGuid = {0x430FD4D0, 0xB729, 0x4F61,
-                            {0xAA, 0x34, 0x91, 0x52, 0x64, 0x81, 0x79, 0x9D}};
+const GUID kGoopdateGuid = {0x1E972C4E, 0x6353, 0x4467,
+                            {0xBF, 0x19, 0x94, 0xDA, 0x11, 0x27, 0xDE, 0xAC}};
 
 // Chrome AppIDs
-#define CHROME_APP_ID _T("{8A69D345-D564-463C-AFF1-A69D9E530F96}")
+#define CHROME_APP_ID _T("{3F5AC3B7-C641-43FD-B7AC-819AF1E0DE38}")
 const TCHAR* const kChromeAppId = CHROME_APP_ID;
```

Comment:
what is that id?

how does client understand what the app to update?

Reply:
It's used for some operations specific to google chrome like appending some argument and displaying specific image, not relevant